nixos/modules/tasks/powertop.nix

```diff
@@ -0,0 +1,27 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.powerManagment.powertop;
+in {
+  ###### interface
+
+  options.powerManagment.powertop.enable = mkEnableOption "Enable powertop auto tuning on startup";
```

Comment:
This will read "Whether to enable Enable powertop auto tuning on startup" in the documentation.